examples/md_ipi/model6.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
```

Comment:
Clean this module, remove things that are not needed (e.g., admp_calculator and read_input_info)